fv3config/fv3run/_docker.py

```diff
@@ -35,59 +36,38 @@ def run_docker(
     """
     if keyfile is None:
         keyfile = os.environ.get("GOOGLE_APPLICATION_CREDENTIALS", None)
-    filesystem.get_fs(outdir).makedirs(outdir, exist_ok=True)
-    with tempfile.NamedTemporaryFile(suffix=".yaml") as config_tempfile:
-        bind_mount_args = []
-        python_args = []
-        docker_args = []
-        config_location = _get_config_args(
-            config_dict_or_location, config_tempfile, bind_mount_args
-        )
```

Comment:
I can't find where config_location is being added to bind_mount_args now, can you point me to it or add it?